scripts/gufi_common.py

```diff
@@ -257,6 +257,17 @@ def build_query(select, tables, where=None, group_by=None,
 
     return query
 
+# a helper function to show the gufi_query being executed. Helpful for debugging and education.
+def print_query(query_tokens):
+  formatted_string = ""
+  for index, token in enumerate(query_tokens):
+      if token.startswith('-') or index == len(query_tokens)-1:
+          formatted_string += '\n\t\t' + token
+      else:
+          formatted_string += ' ' + token
+  print("GUFI query is \n\t{0}".format(formatted_string))
+
+
```

Comment:
extra empty line

Reply:
These two requests done. But still need to address a third request before this PR is ready for you again. Will let you know as soon as I get that.